content/emd/0007-rsa.md

```diff
@@ -239,7 +239,7 @@ $$1203$$
 
 **UPDATE:** Segundo a professora Joana Ventura, podemos passar este passo à frente, a não ser que seja pedido no enunciado.
```

Comment:
Not really related to this PR, but it might be a good idea to remove identifiable information such as teacher's names when it's not relevant